src/main/scala/org/broadinstitute/hail/driver/InfoScore.scala

```diff
@@ -0,0 +1,104 @@
+package org.broadinstitute.hail.driver
+
+import org.apache.spark.rdd.RDD
+import org.apache.spark.storage.StorageLevel
+import org.broadinstitute.hail.utils._
+import org.broadinstitute.hail.annotations.Annotation
+import org.broadinstitute.hail.expr._
+import org.broadinstitute.hail.variant.Genotype
+import org.broadinstitute.hail.variant.Variant
+import org.broadinstitute.hail.variant._
+
+class InfoScoreCombiner extends Serializable {
+  var result = 0d
+  var expectedAlleleCount = 0d
+  var totalDosage = 0d
+  var nIncluded = 0
+
+  def expectedVariance(dosage: Array[Double]): Double = {
+    val mean = dosage(1) + 2 * dosage(2)
+    dosage(1) + 4 * dosage(2) - mean * mean
+  }
+
+  def merge(g: Genotype): InfoScoreCombiner = {
+      g.dosage.foreach { dx =>
+        result += expectedVariance(dx)
+        expectedAlleleCount += dx(1) + 2 * dx(2)
+        totalDosage += dx.sum
+        nIncluded += 1
+      }
+
+    this
+  }
+
+  def merge(that: InfoScoreCombiner): InfoScoreCombiner = {
+    result += that.result
+    expectedAlleleCount += that.expectedAlleleCount
+    totalDosage += that.totalDosage
+    nIncluded += that.nIncluded
+
+    this
+  }
+
+  def thetaMLE = divOption(expectedAlleleCount, totalDosage)
+
+  def imputeInfoScore(theta: Double) =
+    if (theta == 1.0 || theta == 0.0)
+      Some(1d)
+    else if (nIncluded == 0)
+      None
+    else
+      Some(1d - ((result / nIncluded) / (2 * theta * (1 - theta))))
+
+  def asAnnotation: Annotation = {
+    val imputeScore = thetaMLE.flatMap{theta => imputeInfoScore(theta)}
```

Comment:
I think the current code style xml would put spaces around the brackets here.  Can you import those settings and autoformat again?